health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -314,16 +310,17 @@ const FormulaConfiguration = ({ onSelect, category, customProps, formulas: initi
               code: assumptionValue,
               label: assumptionValue,
             })),
-          ]
+          ];
           return (
             <>
               <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)